### PR TITLE
Bug 1220380 - revise pocket menu item in tv

### PR DESCRIPTION
**Pull request by danhuang1202**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5332cf8c8480d6fd6be403a8a4057fc958341fd3

File: tv_apps/browser/style/toolbar.css [hunk 1]
```diff
@@ -705,26 +705,26 @@ input::-moz-focus-inner {
...
   position: absolute;
-  top: 122px;
-  right: -4px;
-  max-width: 1000px;
-  padding: 5px 10px;
-  border-radius: 6px;
+  top: 12.2rem;
+  right: -0.4rem;
+  max-width: 100rem;
+  padding: 1rem 2rem;
```
Comment (begeeben) [inline tv_apps/browser/style/toolbar.css]: Just want to check if the change from 5px to 1rem is by design.

File: tv_apps/browser/style/toolbar.css [hunk 2]
```diff
@@ -741,58 +741,74 @@ input::-moz-focus-inner {
...
-  display: table-cell;
+  display: flex;
   color: #ffffff;
   text-align: left;
-  padding-left: 10px;
-  padding-right: 10px;
-  max-width: 980px;
-  height: 72px;
+  padding-left: 2rem;
+  padding-right: 2rem;
```
Comment (begeeben) [inline tv_apps/browser/style/toolbar.css]: Same as above, just want to confirm that the change from 10px to 2rem is intended.

File: tv_apps/browser/style/toolbar.css (hunk 1 above)
Comment (danhuang1202) [inline tv_apps/browser/style/toolbar.css]: will revert padding to: 0.5rem 2rem;

File: tv_apps/browser/style/toolbar.css (hunk 2 above)
Comment (danhuang1202) [inline tv_apps/browser/style/toolbar.css]: confirmed with UX, this padding should be 2rem

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=2fc7ecf3ff67370134bda1cf79cb8ba3c5273446

**Pull request merged**